litmus-portal/graphql-server/pkg/database/mongodb/operations/project.go

```diff
@@ -77,20 +77,20 @@ func AddMember(ctx context.Context, projectID string, member *dbSchema.Member) e
 	return nil
 }
 
-//RemoveMember ...
-func RemoveMember(ctx context.Context, projectID string, userName string) error {
+//RemoveInvitation ...
+func RemoveInvitation(ctx context.Context, projectID string, userName string) error {
 	query := bson.M{"_id": projectID}
 	update := bson.M{"$pull": bson.M{"members": bson.M{"username": userName}}}
 	_, err := projectCollection.UpdateOne(ctx, query, update)
 	if err != nil {
-		log.Print("Error Removing the member with username:", userName, "from project with project id: ", projectID, err)
+		log.Print(err)
```

Comment:
Prin the above error as `"Error Removing the invite with username:", userName, "from project with project id: ", projectID, err`

Reply:
@Sanjay1611 This is the reason that removeMember and cancelInvite functions should be separate. Although the code is similar for both functions but the use case is different. Separating these functions will also help in easier debugging and code readability in future.

Reply:
Debugging will not be a problem here as by this statement we can know from where the error is happening. For the readability we can add few line comments  on this explaining both the cases @Saranya-jena @SarthakJain26

Reply:
@Sanjay1611 We have added this error message for the case when we will encounter an error removing a member from the project, whereas the one that you suggested will be for the case when we will encounter an error while trying to cancel the invitation for a member.